source/cheby.f90

```diff
@@ -1,41 +1,55 @@
 module cheby
   use parpro
   use floatPrecision
+  use crcoall
+  use mod_alloc
```

Comment:
I would avoid use statements in the header that aren't strictly necessary, as these are inherited when the module is used elsewhere.